src/nrncvode/tqueue.cpp

```diff
@@ -23,94 +20,6 @@
 #define STAT(arg) /**/
 #endif
 
-static const char* errmess_;
-
-static double insert(void* v) {
-    TQueue* q = (TQueue*) v;
-    q->insert(*getarg(1), (void*) 1);
-    return 1.;
-}
-static double print(void* v) {
-    TQueue* q = (TQueue*) v;
-    q->print();
-    return 1.;
-}
-
-static double least(void* v) {
-    TQueue* q = (TQueue*) v;
-    TQItem* i = q->least();
-    double x = -1e9;
-    if (i) {
-        x = i->t_;
-    }
-    return x;
-}
-static double rmleast(void* v) {
-    TQueue* q = (TQueue*) v;
-    TQItem* i = q->least();
-    double x = -1e9;
-    if (i) {
-        x = i->t_;
-        q->remove(i);
-    }
-    return x;
-}
-
-static double mvleast(void* v) {
-    TQueue* q = (TQueue*) v;
-    q->move_least(*getarg(1));
-    return 1.;
-}
-
-static double remove(void* v) {
-    TQueue* q = (TQueue*) v;
-    q->remove(q->find(*getarg(1)));
-    return 1.;
-}
-
-static double find(void* v) {
-    TQueue* q = (TQueue*) v;
-    TQItem* i = q->find(*getarg(1));
-    double x = -1e9;
-    if (i) {
-        x = i->t_;
-        q->remove(i);
-    }
-    return x;
-}
-static double stats(void* v) {
-    TQueue* q = (TQueue*) v;
-    q->statistics();
-    return 1.;
-}
-
-static Member_func members[] = {{"insrt", insert},
-                                {"least", least},
-                                {"move_least", mvleast},
-                                {"remove_least", rmleast},
-                                {"remove", remove},
-                                {"find", find},
-                                {"stats", stats},
-                                {"printf", print},
-                                {0, 0}};
-
-static void* cons(Object*) {
-    assert(0);
-    TQueue* q = new TQueue(0);
-    return (void*) q;
-}
-
-static void destruct(void* v) {
-    TQueue* q = (TQueue*) v;
-    delete q;
-}
-
-void TQueue_reg() {
```

Comment:
When I created this, in the late 90's, I was new to the idea of a priority queue and thought that users would like to explore its behavior. Also that it could be useful for various interpreter algorithms. I think it is unlikely it is being used out in the wild.  WIth all this throwing away of things in the attic, I suppose we'll eventually discard something useful.

Reply:
Oops. Wait a minute. My search for ```Tqueue``` in modeldb.science turned up nothing. But so does ```runworker```. So I guess the search does not search in hoc, python, or mod files.

Reply:
You mean there is some usage I didn't find?

Can I have a link to help me search for my future refactoring?